# pid terms
# original = -50, 0, -5
pid: {
  kp: -50.0,
  ki: -10.0,
  kd: -5.0,
}
# control type
control_type: 'pid'

# turn direction - 'left' or 'right' when the robot is approaching a turn
turn_direction: 'left'

# turn angle - how much to turn wheels during turn. 
# 0 => don't turn at all; 2000 => turn the max angle
# (unsigned int, doesn't change for left/right turns)
turn_angle: 2000

# threshold in meteres when the robot starts turning
turn_threshold: 245

# alpha - the FOV in degree within which the robot will look for a wall
alpha: 60

# normal_speed - the speed with which robot will normally drive (6400 original)
#(6400 for racing)
#(6250 for stop detection)
normal_speed: 6400

# turn_speed - the speed with which robot will drive while turning at corner
turn_speed: 6150

# enable_stop_sign - whether we should look for stop signs to stop at
enable_stop_sign: false

# the speed for which it goes immediately after the turn 
#(6400 for racing)
#(6250 for stop detection)
turn_exit_speed: 6400

# slow_threshold -  threshold in meters when the robot starts slowing down
slow_threshold: 150

turn_yaw_threshold: .25

inlet_threshold: 2.0

run_avg_alpha: 0.5
